mocha-client-tests/karma.conf.js

```diff
@@ -0,0 +1,72 @@
+module.exports = function(config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['mocha'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      // Test dependencies
+      'node_modules/expect.js/index.js',
+      'node_modules/sinon-chrome/bundle/sinon-chrome-webextensions.min.js',
```

Comment:
@kumar303 @rpl: thanks to you both, that makes sense. I'll take a quick look at adding Promise support to sinon-chrome for methods with async properties.